crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs

```diff
@@ -73,16 +73,16 @@ impl Rule for NoAutofocus {
     fn from_configuration(value: serde_json::Value) -> Self {
         let mut no_focus = Self::default();
 
-        let _ = value.as_array().unwrap().iter().find(|v| {
+        if value.as_array().unwrap().iter().any(|v| {
```

Comment:
Won't this also panic if it's now an array?

Reply:
Ooh, right, I noticed that the config can only a number like this :
```
{
  "rules": {
    "jsx-a11y/rule-name": 2
  }
}
```
or just an object:
```
{
  "rules": {
    "jsx-a11y/rule-name": {
            "ignoreNonDOM": true
        }
  }
}
```

Reply:
Maybe a public method should be distilled here 🤔